spec/20-http_request_header_format.md

```diff
@@ -300,6 +300,8 @@ lcalpha    = %x61-7A ; a-z
 
 A `key` MUST begin with a lowercase letter or a digit and contain up to 256 characters including lowercase letters (`a`-`z`), digits (`0`-`9`), underscores (`_`), dashes (`-`), asterisks (`*`), forward slashes (`/`), and at signs (`@`).
 
+This restriction ensures that the `tracestate` header can be parsed efficiently into individual list members by splitting the header value at each comma, and splitting the list members an implementation is interested in at the equals sign.
```

Comment:
As noted in my [comment](https://github.com/w3c/trace-context/issues/508#issuecomment-1335454580), this reasoning is incorrect. Nothing about Unicode text, particularly UTF-8, is less-efficient for parsing, including looking for comma/equals signs.

While there really is no *good* reason (from an I18N point-of-view) for this restriction, I understand that it *feels* simpler to restrict to ASCII. Thinking about it at length, I'd suggest the following instead:

> This restriction on the `tracestate` header ensures that even very low-level and simple computing devices can process, validate, and display headers without having to worry about mid-character truncation or certain classes of display issues due to the use of e.g. bidi controls.

Reply:
I understand why you say that the phrase I added is actually incorrect, even if that (flawed) thinking might have been the original reason for the design choice to limit the character set the way we do. 

But to be honest I'm skeptical about your suggested phrasing as well:
* When you say there is "no good reason" for this restriction, we don't need to make up a reason. We already [established](https://github.com/w3c/trace-context/issues/508#issuecomment-1333468843) that by now the main reason to keep this restriction is backwards compatibility.  
* _Displaying_ values from the header to users is not a use case this spec is concerned with. The [Problem Statement](https://www.w3.org/TR/trace-context/#problem-statement) and [Solution](https://www.w3.org/TR/trace-context/#solution) section clearly call out what this spec is about: Trace context propagation. The trace context is of no concern to users, it is a mechanism for observability tooling. Displaying individual key-value pairs to users might be something that is relevant for the baggage spec but not for this spec.

I see two ways forward here:
* We leave this unchanged and do not add any reasoning at all. Yes, the restriction is no ideal from an i18n point of view, but it cannot be easily changed now. Adding a reasoning doesn't change that. Adding a partially made up reason just for the sake of adding a reason doesn't add value IMO.
* If you insist on us adding a reasoning here, I think we could meet halfway with either of those phrasings: 

> This restriction on the `tracestate` header ensures that even low-level computing devices can process and validate it.

or

> This restriction on the `tracestate` header ensures that even low-level computing devices can process and validate it, without having to worry about mid-character truncation.

Reply:
Actually, I don't insist that you add a reason--just that, if you do have a reason, that it make sense.

I don't buy that the header values are not displayed. While this spec may not be concerned with the display of values, you do need to think about the larger trace ecosystem. The keys have meaningful structure, presumably because people will use them for various things. Otherwise you could just generate completely opaque values.

I would suggest that you just drop this PR? I'd be satisfied with that.

Reply:
Yep. And I'll add a _satisfied_ comment to that issue when you do.

Reply:
👍

Reply:
@aphillips done.